x/config/websocket.go

```diff
@@ -0,0 +1,134 @@
+// Copyright 2024 Jigsaw Operations LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net"
+	"net/url"
+	"strings"
+
+	"github.com/Jigsaw-Code/outline-sdk/transport"
+	"golang.org/x/net/websocket"
+)
+
+type wsConfig struct {
+	tcp_path string
+	udp_path string
```

Comment:
[Go style recommends `mixedCaps` rather than underscores](https://google.github.io/styleguide/go/guide#mixedcaps).

```suggestion
	tcpPath string
	udpPath string
```

Reply:
Fixed